### ISSUE TITLE
Generator breaks jQuery plugins requires at application.js

### ISSUE THREAD

**Issue by szyablitsky** [shakacode/react_on_rails#117]
```
//= require jquery-ui.min
//= require jquery.fileupload
```
becomes
```
-ui.min
.fileupload
```

**Comment by robwise** [comment]
Because we're importing jQuery from Webpack, we had programmed the generator to search for and delete the normal jquery sprocket require statements via gsub. I suspect the gsub regex is too loose and is picking up these even though it shouldn't. 

However, and @justin808 can confirm this, I believe that if you want to use JavaScript libraries, it would be best practice to use their NPM versions by including them in your `client/package.json` file.

Either way, this still should be addressed.

**Comment by justin808** [comment]
@szyablitsky Any chance that you can propose a PR for this?

@robwise yes, your match is too aggressive, and it should comment out, not delete, existing

**Comment by szyablitsky** [comment]
I will do the PR tomorrow.

**Comment by marcellosachs** [comment]
I had a similar issue - my application.js file now contains the line 'ui.custom'

**Comment by justin808** [comment]
Thanks. Let's try to get this fix in the 1.1.0 release or shortly thereafter.

**Comment by szyablitsky** [comment]
@justin808 PR ready.
